app/Http/Controllers/Auth/ApiTokenController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\ApiTokens\ApiTokenStoreRequest;
+use App\Repositories\ApiTokenRepository;
+use Exception;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Str;
+use Illuminate\View\View;
+
+class ApiTokenController extends Controller
+{
+    /** @var ApiTokenRepository */
+    private $apiTokensRepo;
+
+    public function __construct(ApiTokenRepository $apiTokensRepo)
+    {
+        $this->apiTokensRepo = $apiTokensRepo;
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function index(): View
+    {
+        $tokens = $this->apiTokensRepo->all(auth()->user()->currentWorkspaceId());
+
+        return view('api-tokens.index', compact('tokens'));
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function store(ApiTokenStoreRequest $request): RedirectResponse
+    {
+        $input = $request->validated();
+
+        $newToken = Str::random(32);
+
+        $this->apiTokensRepo->store(
+            auth()->user()->currentWorkspaceId(),
```

Comment:
@dljfield I wonder if we should always rely on `Sendportal::currentWorkspaceId()` because the resolver can have multiple implementations and at the moment, API already would use something else:
```
        Sendportal::currentWorkspaceIdResolver(function() {
            if (auth()->user()) {
                return auth()->user()->currentWorkspaceId();
            }

            if ($apiToken = request()->bearerToken()) {
                return ApiToken::resolveWorkspaceId($apiToken);
            }
        });
```

Reply:
Yeah, good idea. Have done that.